validator/src/bin/solana-test-validator.rs

```diff
@@ -292,6 +292,11 @@ fn main() {
                 .hidden(true)
                 .help("Specify the configuration file for the AccountsDb plugin."),
         )
+        .arg(
```

Comment:
This is changing the current default behavior.  Maybe it makes sense to use accounts_db_caching_enabled which is default to false. If it is turned on -- caching can be enabled.

Reply:
Indeed. Should I disable it by default?
But the more important question should account notifications work without enabled/disabled caching?

Reply:
It is only designed to be working with caching enabled -- the disabled path will never be able to catch up with the network.

Reply:
@mvines are we ok with enabled caching by default in test-validator?

Reply:
Whatever @lijunwangs things makes sense, I don't know anything about the accounts db plugin stuff

Reply:
No, it's not about accounts db, sorry if a message doesn't look clear :disappointed: 
This PR changes `solana-test-validator` behavior by default but at the same time makes behavior consistent with `solana-validator`. Should we change `solana-test-validator` config to be consistent with `solana-validator` or let it be as it is?

Reply:
Consistency by default makes sense to me. So this PR is good as is then?

Reply:
Yes. PR makes caching option behavior the same in validator and in test-validator.

Reply:
It makes sense to make test-validator behavior consistent with the validator in production.

Reply:
👍🏼 wfm